### PR TITLE
ItemDB Updates:

### PR DESCRIPTION
**Pull request by Jedzkie**
* [x] Implement official effect of MVP Scrolls.
* [x] Change ID to Constants
* [x] Dropping 'SC_ITEMSCRIPT' (SC_ITEMSCRIPT is not present in aegis script commands and status list)

### CODE REVIEW

**Comment by Jedzkie** [comment]
@MishimaHaruna @hemagx @dastgir @AnnieRuru 

I need help guys, SC_MVPCARD_ORCLORD behavior is kinda crushing my brain, I dont know how to code this.. xD

File: src/map/battle.c [hunk 1]
```diff
@@ -5817,6 +5817,17 @@ void battle_reflect_damage(struct block_list *target, struct block_list *src, st
...
 					delay += 100;/* gradual increase so the numbers don't clip in the client */
 				}
+				if (sc->data[SC_MVPCARD_ORCLORD]) {
+					NORMALIZE_RDAMAGE(damage * sc->data[SC_MVPCARD_ORCLORD]->val1 / 100);
+
+					rdelay = clif->delay_damage(tick + delay, src, src, status_get_amotion(src), status_get_dmotion(src), rdamage, 1, BDT_ENDURE);
+
+					if (tsd)
+						battle->drain(tsd, src, rdamage, rdamage, status_get_race(src), 0);
+						battle->delay_damage(tick, wd->amotion, target, src, 0, CR_REFLECTSHIELD, 0, rdamage, ATK_DEF, rdelay, true);
```
Comment (AnnieRuru) [inline src/map/battle.c]: remove 1 indent space here
when I see you indent this way, I actually thought you forgot to add curly bracket

**Comment by Jedzkie** [comment]
@AnnieRuru Done.

**Comment by AnnieRuru** [comment]
tested ok now

**Comment by dastgir** [comment]
Seems fine?
Anything remaining in this?
Also, please squash the follow-up commit

**Comment by MishimaHaruna** [comment]
Merged, thanks!

**Pull request closed without merging**